testhelper/certificatetest.go

```diff
@@ -31,6 +36,11 @@ type ECCertTuple struct {
 	PrivateKey *ecdsa.PrivateKey
 }
 
+type ED25519CertTuple struct {
+	Cert       *x509.Certificate
+	PrivateKey *ed25519.PrivateKey
+}
```

Comment:
Where is ed25519 used? It is not in the support list of notary.

Reply:
Removed